dev/CommandBarFlyout/CommandBarFlyout.cpp

```diff
@@ -252,25 +248,129 @@ winrt::Control CommandBarFlyout::CreatePresenter()
     presenter.BorderThickness(winrt::ThicknessHelper::FromUniformLength(0));
     presenter.Padding(winrt::ThicknessHelper::FromUniformLength(0));
     presenter.Content(*commandBar);
-    // Clear the default CornerRaius(4) on FlyoutPresenter, CommandBarFlyout will do its own handling.
+
+    // Disable the default shadow, as we'll be providing our own shadow.
+    if (winrt::IFlyoutPresenter2 presenter2 = presenter)
+    {
+        presenter2.IsDefaultShadowEnabled(false);
+    }
```

Comment:
This code previously had a check for Cobalt.  Is that no longer needed?  Please test downlevel as well.

Reply:
It's not needed anymore. Originally this setting was set to false only when pre-Cobalt because we were directly handling the projected shadows, and we wanted to use the default shadow for drop shadows. Now that we'll need to manage the shadows for Cobalt as well, we no longer need the Cobalt check.

Reply:
Thanks for the explanation!

---
In reply to: [612005543](https://github.com/microsoft/microsoft-ui-xaml/pull/4743#discussion_r612005543) [](ancestors = 612005543)